zerver/migrations/0423_fix_attachment_owner.py

```diff
@@ -0,0 +1,172 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.db import connection, migrations
+from django.db.backends.postgresql.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from psycopg2.sql import SQL, Identifier, Literal
+
+
+def fix_attachment_owner(apps: StateApps, schema_editor: BaseDatabaseSchemaEditor) -> None:
```

Comment:
Might be worth calling this file `fix_email_gateway_attachment_owner`, for clarity of its scope.